tools/syntax/VsCode/syntaxes/bond.tmLanguage

```diff
@@ -0,0 +1,168 @@
+<plist version="1.0">
+<dict>
+    <key>fileTypes</key>
+    <array>
+        <string>bond</string>
+    </array>
+    <key>name</key>
+    <string>Bond</string>
+    <key>patterns</key>
+    <array>
+        <dict>
+            <key>include</key>
+            <string>#commonStuff</string>
+        </dict>
+        <dict>
+            <key>captures</key>
+            <dict>
+                <key>1</key>
+                <dict>
+                    <key>name</key>
+                    <string>keyword.other.namespace.source.bond</string>
+                </dict>
+                <key>2</key>
+                <dict>
+                    <key>name</key>
+                    <string>storage.type.bond</string>
+                </dict>
+                <key>3</key>
+                <dict>
+                    <key>name</key>
+                    <string>entity.name.type.namespace.source.bond</string>
+                </dict>
+            </dict>
+            <key>match</key>
+            <string>^(namespace)\s+(cpp|csharp) (.+)$</string>
+            <key>name</key>
+            <string>meta.namespace.source.cs</string>
+        </dict>
+        <dict>
+            <key>match</key>
+            <string>^import\s+</string>
+            <key>name</key>
+            <string>keyword.other.import.source.bond</string>
+        </dict>
+        <dict>
+            <key>begin</key>
+            <string>(enum)\s+(.+)$</string>
+            <key>captures</key>
+            <dict>
+                <key>1</key>
+                <dict>
+                    <key>name</key>
+                    <string>keyword.other.enum.source.bond</string>
+                </dict>
+                <key>2</key>
+                <dict>
+                    <key>name</key>
+                    <string>storage.type.source.enum.bond</string>
+                </dict>
+            </dict>
+            <key>end</key>
+            <string>\}</string>
+            <key>name</key>
+            <string>meta.enum.source.bond</string>
+            <key>patterns</key>
+            <array>
+                <dict>
+                    <key>include</key>
+                    <string>#commonStuff</string>
+                </dict>
+                <dict>
+                    <key>captures</key>
+                    <dict>
+                        <key>1</key>
+                        <dict>
+                            <key>name</key>
+                            <string>entity.name.bond</string>
+                        </dict>
+                    </dict>
+                    <key>match</key>
+                    <string>^\s*([^\s]*)\s</string>
+                    <key>name</key>
+                    <string>entity.name.type.enum.value.bond</string>
+                </dict>
+            </array>
+        </dict>
+        <dict>
+            <key>begin</key>
+            <string>(struct)\s+(.+)</string>
```

Comment:
Can I add this in a future drop?  It's currently all colored the same (which isn't great, but not _terrible_)